README.rst

```diff
@@ -333,6 +340,34 @@ Modifier
 
     - .get_modifier()
 
+Exploding
+    A die that continues rolling if the maximum value is rolled, adding the previous roll to the results.
```

Comment:
not clear, too long

An exploding die is a die that has a chance to roll again.